### ISSUE TITLE
Make AJ37 or AJS37 variant

### ISSUE THREAD

**Issue by NikolaiVChr** [NikolaiVChr/flightgear-saab-ja-37-viggen#37]
@l0k1 is working on the AJ model. There are some issues to sort out before this can be done though.

**The engine issue**

The picture below details the RM-8A data from the AJS-37 manual:

![rm8a](https://cloud.githubusercontent.com/assets/3764194/15949703/ca03f25c-2ea9-11e6-992f-636ffad9ad6c.png)

What is strange is that the engine data of the RM-8B that should be more powerful, is even mentioned in the JA-37Di manual as seen below, is actually rated lower than the RM-8A. This is strange.

![rm8b](https://cloud.githubusercontent.com/assets/3764194/15949706/ce682ad4-2ea9-11e6-9b7c-94d36443327e.png)
A/T = Auto-throttle

What is also strange is that every online reference and all the books about the Viggen gives these values for performance:

**RM8A**
mil = 65,6 kN
aug = 115,6 kN

**RM8B**
mil = 72,1 kN
aug = 125,0 kN

These values match the RM8A fine, but they are way higher for RM8B than what the manuals for JA-73C and JA-37Di state.

Another issue: Why is the above performance values not shown for 100% N2? The RPM for 100% is shown, so the engines must be able to do 100%, so why is the max performance quoted at 97.3% and 94.8%??

**Comment by NikolaiVChr** [comment]
**Misc stuff to differ**

Store loadout.
Radar range.
3D Model
GUI menu name
GUI help text
HUD symbology
Emptyweight

**Comment by l0k1** [comment]
For what it's worth, the AJ-37 model is complete (it can also double as an SH-37 model, I believe).

I am working on armament - I plan on the full suite of armament to be modeled as well. Once this is complete I'll submit the models.

One other issue that might pop up is re-working the gun code to accept 1 or 2 ADEN pods.

**Comment by NikolaiVChr** [comment]
Sounds great Pinto, this project is starting to give me headache though, researching fuel and engine.

I just found a leaflet from Saab (on my HD of all places).

It confirms the high numbers for the JA-37, which means it is underpowered in FG.

Why the hell does the JA manuals lie to the pilots...the AJS-37 pilots gets the right info...grrrrr

**Comment by l0k1** [comment]
Haha because they can't make it easy!

BK90 Mjolner is done - not going to do submunition per our discussion.

On the todo list is...

- ADEN gunpods
- RB04
- RB05
- RB75
- M71

**Comment by NikolaiVChr** [comment]
Of the stuff I need to do:

- [ ] JSBsim refactored into AJ and JA variant, including different engine and fuel contents.
- [ ] Welcome message changed
- [ ] Splashscreen for AJ37
- [ ] Set file and model file refactored into variants.
- [ ] Export the new AJ and JA model files.
- [ ] GUI menu and help refactored into variants.
- [ ] Different radar range

**Comment by l0k1** [comment]
Add to that todo list...

- [ ] Disable Oerlikon

=]

**Comment by NikolaiVChr** [comment]
Okay I listed them here what you mentioned. If there is any mistakes please tell.

- [ ] RB-24 - Sidewinder
- [ ] RB-24J Sidewinder
- [ ] M70 - Raketkapsel (364 kg) <-- weight to be changed in FG
- [ ] RB-15F - Robot 15 (598 kg)
- [ ] BK90 - Bombkapsel "Mjölner"
- [ ] M55 AKAN "ADEN" (364 kg)
- [ ] RB04E - Robot 04 (620 kg)
- [ ] RB05A - Robot 05 (305 kg)
- [ ] RB75 - Maverick (210 kg)
- [ ] M71 type 1 - Bomblavett "Virgo" (4 x 120 kg M63FFV explosive)
- [ ] M71 type 2 - Bomblavett "Virgo" (4 x 80 kg lightbomb)

**Comment by NikolaiVChr** [comment]
Lot of work, especially M71 those need a new type of firecontrol system I think.

**Comment by NikolaiVChr** [comment]
Mjölner Data:

**Source 1** http://www.x-plane.org/home/urf/aviation/text/mjolner.htm

Length 3.505 m
Span   1.000 m  
Width  0.630 m
Height 0.595 m, fuselage 0.320 m 
Weight  600 kg  

Release altitude 30 - 500 m
Release speed M 0.6 - M 0.9
Flight altitude 30 - 200 m, terrain following
Range when launched at 50 m altitude and Mach 0.9 is more than 10 km in the direction the aircraft is flying

**Source 2** http://cmano-db.com/aircraft/533/

BK 90 Mjolner Mk1 [72 x MJ1 Anti-Personnel] - (DWS.39) Guided Weapon
Land Max: 14.8 km.

**Comment by NikolaiVChr** [comment]
AKAN Data:

**Source 3** https://sv.wikipedia.org/wiki/30_mm_automatkanon_m/55

Vapentyp	Automatkanon
Kaliber	30 mm
Piplängd	1,08 m
Ammunition	30 mm x 113
Låsmekanism	Revolverkanon
Längd	1,59 m
Vikt	87,1 kg
Eldhastighet	1200-1700 rpm
Mynningshastighet	741 m/s

**Source 2** http://cmano-db.com/aircraft/533/

30mm ADEN Mk4 x 2 Burst [60 rnds] - (Aircraft) Gun
Air Max: 1.9 km. Surface Max: 2.8 km. Land Max: 2.8 km.

**Comment by NikolaiVChr** [comment]
RB04E data:

**Source 4** https://sv.wikipedia.org/wiki/Robot_04

Längd	4 453 mm
Vikt	625 kg
Spännvidd	1944 mm
Diameter	500 mm
Stridsspets	trotyl
Stridsspetsvikt	300 kg
Tändrör	zonrör
Motor	IMI Summerfield raketmotor
Brinntid	43 s (04C)
65,6 s (04D, 04E)
Målsökare	aktiv radar
Vapenbärare	A 32 Lansen
AJ 37 Viggen
Prestanda
Räckvidd	32 km
Flyghöjd	10–15 meter
Hastighet	underljudsfart

**Comment by NikolaiVChr** [comment]
RB05A data:

**Source 5** https://sv.wikipedia.org/wiki/Robot_05

Längd	3,6 meter
Vikt	305 kg
Spännvidd	0,8 meter
Diameter	0,3 meter
Stridsspetsvikt	160 kg
Tändrör	Anslagsrör
Motor	Vätskeraketmotor
Målsökare	MCLOS via radiolänk
Vapenbärare	AJ 37, AJS 37
Prestanda
Räckvidd	9 km
Maxhastighet	supersonisk

**Comment by NikolaiVChr** [comment]
RB75 data:

Is probably AGM-65B version, which were developed in 1975 and Sweden ordered them that year. 

Diameter	30 cm (12 in)
Weight	210 kg (462 lb)
Speed	1,150 km/h (620 kn)
Range	Greater than 22 km (12 nmi)
Warhead	57 kg (126 lb) WDU-20/B shaped-charge
Supersonic

**Comment by NikolaiVChr** [comment]
So I am thinking about making the HUD/loadout symbology consistent.

I think in the HUD there should be either RB-XX, M?? or AKAN (whether its pods or center mounted). And drop the KCA symbol.

In the loadout, I think it should be RB-XX or M??.

Or do you think we should say ARAK for rocketpods M70.

We could also change the loadout names to include both number and type, like:

M70 ARAK
RB-24 Sidewinder
RB-75 Maverick
M55 AKAN
M90 Bombkapsel

Or could flesh it out even more, and write info in english:

M70 ARAK Rocketpod
RB-24 Sidewinder A/A
RB-75 Maverick A/G
M55 AKAN Cannonpod
M90 Bombkapsel Clusterbombs

But that will maybe make it seem more gamey and less sim like. Not sure..

**Comment by NikolaiVChr** [comment]
As per our discussion:

RB 04E Attackrobot
RB 05A Attackrobot
RB 15F Attackrobot
RB 24 Sidewinder
RB 24J Sidewinder
RB 71 Skyflash
RB 74 Sidewinder
RB 75 Maverick
RB 99 Amraam
M55 AKAN
M70 ARAK
M71 Bomblavett
M90 Bombkapsel

**Comment by NikolaiVChr** [comment]
Just noticed picture on page 7 of this document. Notice the fins on the droptank:

https://drive.google.com/open?id=0B49qitQsNy34RnZISk9TWHcxVkU

**Comment by l0k1** [comment]
But what about page 5!?!?!

**Comment by l0k1** [comment]
It'd be easy to add in that top fin to the AJ drop tank

**Comment by NikolaiVChr** [comment]
Some do not seem to fit:

![fgfs-screen-402](https://cloud.githubusercontent.com/assets/3764194/16064741/41a60b62-32a4-11e6-957f-5c1ea39c8ac8.png)

**Comment by NikolaiVChr** [comment]
I regard this issue as fixed.
